src/main/java/seedu/address/logic/commands/quizcommands/ViewScoreCommand.java

```diff
@@ -1,32 +1,38 @@
 package seedu.address.logic.commands.quizcommands;
 
+import seedu.address.commons.core.index.Index;
 import seedu.address.logic.commands.Command;
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.logic.commands.exceptions.CommandException;
-import seedu.address.model.QuizModel;
+import seedu.address.model.Model;
+import seedu.address.model.flashcard.FlashcardSetName;
 
-public class ViewScoreCommand extends Command<QuizModel> {
+public class ViewScoreCommand extends Command<Model> {
```

Comment:
This is what I was worried about, when I was refactoring, I found out that the QuizModel cannot really be separated due to the strong coherence with the Flashcard.

Would you think adding one more layer of the Model (QuizFlashcardModel) would be a better structure??

Reply:
Yeah I agree, we can do that